solutions/standard/main.tf

```diff
@@ -232,3 +232,83 @@ data "ibm_database_connection" "existing_connection" {
   user_id       = data.ibm_database.existing_db_instance[0].adminuser
   user_type     = "database"
 }
+
+########################################################################################################################
+# Code Engine Kibana Dashboard instance
+########################################################################################################################
+
+data "http" "es_metadata" {
+  count    = var.enable_kibana_dashboard ? 1 : 0
+  url      = "https://${local.es_username}:${local.es_password}@${local.es_host}:${local.es_port}"
+  insecure = true
+}
+
+locals {
+
+  code_engine_project_name = var.code_engine_project_name != null ? var.code_engine_project_name : var.prefix != null ? "${var.prefix}-code-engine-kibana-project" : "ce-kibana-project"
+  code_engine_app_name     = var.prefix != null ? "${var.prefix}-kibana-app" : "ce-kibana-app"
+
+  es_host         = local.use_existing_db_instance ? data.ibm_database_connection.existing_connection[0].https[0].hosts[0].hostname : module.elasticsearch[0].hostname
+  es_port         = local.use_existing_db_instance ? data.ibm_database_connection.existing_connection[0].https[0].hosts[0].port : module.elasticsearch[0].port
+  es_username     = local.use_existing_db_instance ? data.ibm_database.existing_db_instance[0].adminuser : "admin"
+  es_password     = local.admin_pass
```

Comment:
For this to work on an existing ES instance, the user would have to make sure they pass values for `var.existing_db_instance_crn` and `var.admin_pass`. I think you need to add validation to ensure this when user sets `enable_kibana_dashboard` to true

Reply:
I was going to say to ensure to deploy kibana in an existing resources test, but it seems we don't have one! This is a gap for sure, and I see we have an issue for it: https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/issues/282

I bumped the priority of our internal issue to add this

Reply:
> For this to work on an existing ES instance, the user would have to make sure they pass values for `var.existing_db_instance_crn` and `var.admin_pass`. I think you need to add validation to ensure this when user sets `enable_kibana_dashboard` to true

if password is not provided, it will automatically create one.